Bug: https://github.com/rust-lang/rust/issues/67242
Description: this test is new in 1.40 and the failure is present on many
 previous rust versions without users having complained or noticed, so
 just ignore the failure until upstream fixes it
--- a/src/test/run-make-fulldeps/foreign-exceptions/Makefile
+++ b/src/test/run-make-fulldeps/foreign-exceptions/Makefile
@@ -1,3 +1,5 @@
+# ignore-arm
+
 -include ../tools.mk
 
 all: foo
